package.json

```diff
@@ -166,6 +166,7 @@
     "no-ui-slider": "1.2.0",
     "node-fetch": "1.3.2",
     "pegjs": "0.9.0",
+    "phantomjs-prebuilt": "^2.1.16",
```

Comment:
What is this change for?

Reply:
I was getting some errors while running yarn so i removed some package and added it again so that change is being reflected here.